ApiBundle/Exceptions/HttpException/TransformParameterTypeHttpTypeHttpException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Exceptions;
+
+/**
+ * Class TransformerParameterException
+ */
+class TransformerParameterTypeHttpException extends \Exception
```

Comment:
This exception should extent `OpenOrchestra\BaseApiBundle\Exception\ApiException`